root/etc/e-smith/templates/etc/yum/vars/nsrelease/00default

```diff
@@ -8,7 +8,7 @@
        <$nsh>;
    };
 
-   our $release = '';
+   our $release = $defaultRelease;
```

Comment:
Remove `/etc/yum/vars/nsrelease` template and add the standard config file inside nethserver-release.

Reply:
if we remove that template, the automatic upgrade of subscriptions must be implemented by an action